.agents/skills/autoship/SKILL.md

```
@@ -0,0 +1,216 @@
+5. **Branch** - Creates release branch with changeset file
+6. **PR** - Creates pull request for the changeset
+7. **Wait** - Waits for CI checks to pass
+8. **Merge** - Merges the changeset PR
+9. **Version PR** - Waits for changesets action to create Version Packages PR
+10. **Publish** - Merges Version Packages PR to trigger npm publish
+
+## Output Format
+
+autoship provides clear step-by-step output:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)